react/Radio/Readme.md

```diff
@@ -65,22 +66,23 @@ const Box = ({ children }) => {
 
 const bboxStyle = { border: '1px solid red' }
 
-initialState.bbox = isTesting();
+initialState.bbox = isTesting()
+
+;
 
 <>
   <p>
-    <Radio checked={state.bbox} onClick={() => setState({ bbox: !state.bbox })} label="View bounding box" />
+    <Checkbox checked={state.bbox} onClick={() => setState({ bbox: !state.bbox })} label="View bounding box" />
```

Comment:
Here we replace Radio by Checkbox. I don't understand why with the commit message?

Reply:
j'ai ajouté du commentaire dans le commit, ici on a un comportement checkbox le radio n'est pas approprié